### ISSUE TITLE
Split code into different modules

### ISSUE THREAD

**Issue by wintlu** [strukturag/libde265.js#12]
Currently we have a huge big source file, it is difficult to read and maintain. Is it possible we split it into different modules (maybe as a npm module)?
I am asking this because I only need the Decoder feature of this library. I am using webpack as build tool

**Comment by adminy** [comment]
Its simple and easy to do. I've done it if you need it. But I don't see how it makes it easier to maintain? Big source file meaning all in 1 place. Its in separate functions already so the code looks very clear, except the Enscripten Magic ...

**Comment by hotkidfamily** [comment]
did you get it ?

**Comment by adminy** [comment]
@hotkidfamily yes, long ago. It takes about 2 days to do. But really it’s mostly already separated into modules. Have a look at the source code.
